geogem.rb

```diff
@@ -119,4 +119,4 @@ def point_buff(r,steps)
 end
 
 
-#Polygon.new("0 0, 2 2, 2 0, 0 2, 0 0").self_intersects?
\ No newline at end of file
+Polygon.new("0 0, 10 0, 10 10, 0 10, 0 0").intersects?(Polygon.new("5 5, 20 0, 20 30, 0 20, 5 5"))
```

Comment:
What's the purpose of this line?

Reply:
ahh, more testing / debugging stuff hanging around :-1: